src/test/java/organization/OrganizationsTest.java

```diff
@@ -137,9 +154,101 @@ public void updateOrganization() {
     }
 
```

Comment:
Zastanawiam się czemu nie zrobiłeś tego samego co wyżej? Czemu nie przygotowałeś sobie niepoprawnych danych w metodzie z danymi i nie zrobiłeś do tego jednego testu no createOrganizationWithInvalidData? Odnośnie nazywanie testów to jest kilka szkół, jedna jest taka, że opisujesz co robisz i co powinno zostać zwrócone no createOrganizationWithInvalidDataShouldReturnStatusCode400 - trochę długa ta nazwa ale wszystko widać.